x-pack/plugins/cases/public/components/create/form_context.tsx

```diff
@@ -142,7 +144,20 @@ export const FormContext: React.FC<Props> = ({
           ? normalizeActionConnector(caseConnector, fields)
           : getNoneConnector();
 
-        const transformedCustomFields = transformCustomFieldsData(customFields);
+        const configurationOwner: string = selectedOwner ? selectedOwner : owner[0];
+        const selectedConfiguration = allConfigurations.find(
+          (element: CasesConfigurationUI) => element.owner === configurationOwner
+        );
+        const backupConfiguration = allConfigurations[0];
```

Comment:
Just to verify, the `allConfigurations` will always have a configuration because if the backend will not return anything the hook will return `[initialConfiguration]`, correct?

Reply:
Also, I don't think we should default to the first one we find. What if the first configuration is on a different owner?

Reply:
Let's discuss this offline